### ISSUE TITLE
Install and implement MUI

### ISSUE THREAD

**Issue by David5886** [cs481-ekh/f22-fires-wild#243]

**Comment by David5886** [comment]
11/5
Spent about 6 hours working on this task. About an hour and a half trying to figure out how to install MUI. Then the rest of the time was reading documentation and figuring out how to use it. 

I was able to redo our navigation bar. I think it looks nicer. Might make more adjustments based on feedback from teammates. 

I also started implementation of dark mode! Right now it's set as default, but I want light to be default and have a toggle to toggle between the two themes.

**Comment by David5886** [comment]
11/6
I spent about another 7-8 hours working on this today. Yesterday I did the navbar and started working on a potential dark mode. Today I spent time working on making the UI for our data page much more nicer to look at. The biggest hurdles I faced was I couldn't just change the input types and call it a day. I first implemented sliders for our DOY variables. That took a while since I couldn't copy and paste. I had to go through each element and add it to the slider. A big thing is that sliders use inputProps while as reacts numericinputs do not. I had many issues with the site crashing or not loading whenever I tried implementing these sliders. Then I needed to update the input boxes for fire size to use MUIs input boxes. That took a while as I was trying to copy and paste from the sliders. But ran into issues with trying to layout everything. The layout took me a while to get right. Mostly from just trial and error. 
We decided to not use sliders for fire size for now, as we would need to create another endpoint to do so.
